src/test/java/com/rtb/projectmanagementtool/auth/AuthOpsTest.java

```diff
@@ -54,18 +43,35 @@ public void tearDown() {
 
   @Test
   public void testLoginUser() {
+    // add list of mock users
+    ArrayList<UserData> testUsers = new ArrayList<UserData>();
+    testUsers.add(new UserData(1l, "abc"));
+    testUsers.add(new UserData(2l, "opq"));
+    testUsers.add(new UserData(3l, "xyz"));
+
     // new controller
     UserController controller = mock(UserController.class);
-    controller.addUser(testUser);
 
     // build and send a test cookie
-    cookie = mock(Cookie.class);
-    cookie.setName("sessionUserID");
-    cookie.setValue("out");
+    Cookie[] testCookies = new Cookie[3];
+    testCookies[0] = new Cookie("SessionUserID", "abc");
+    testCookies[1] = new Cookie("SessionUserID", "zvm");
+    testCookies[2] = new Cookie("SessionUserID", "wpk");
```

Comment:
If we're trying to test the login scenario, we would want this to look like there was no user that's logged in.  Either don't set SessionUserId, or set it to the NO_USER_LOGGED_IN value (which we were discussing could be zero or -1).